packages/components/src/tools-panel/styles.ts

```diff
@@ -145,3 +145,19 @@ export const ToolsPanelItemPlaceholder = css`
 export const DropdownMenu = css`
 	min-width: 200px;
 `;
+
+export const DefaultControlsItem = css`
+	span:last-child {
+		color: var( --wp-admin-theme-color-darker-10 );
+		margin-left: ${ space( 3 ) };
+		${ baseLabelTypography }
```

Comment:
Not super important, but I'm mildly cautious about using this mixin here because it's not the intended use case. What makes sense as a style change for the intended use case may not make sense in this menu item context. I'm fine with copy-pasting the actual styles if that makes more sense.

Reply:
Agreed. I've copied the actual styles into the new styled component for the reset text: https://github.com/WordPress/gutenberg/pull/44260/commits/dfd7e8bf66a018d3a232297bd241f98c201e0ab4

Reply:
Thank you!